bin/ohi

```diff
@@ -85,23 +92,27 @@ class Ohi(object):
 
         parser = argparse.ArgumentParser(description='OpenShift Host Inventory')
 
-        parser.add_argument('--list-host-types', default=False, action='store_true',
-                       help='List all of the host types')
+        parser.add_argument('--list-host-types', default=False, action='store_true', help='List all of the host types')
 
-        parser.add_argument('-e', '--env', action="store",
-                       help="Which environment to use")
+        parser.add_argument('-e', '--env', action="store", help="Which environment to use")
 
-        parser.add_argument('-t', '--host-type', action="store",
-                       help="Which host type to use")
+        parser.add_argument('-t', '--host-type', action="store", help="Which host type to use")
 
-        parser.add_argument('-l', '--user', action='store', default=None,
-                               help='username')
+        parser.add_argument('-l', '--user', action='store', default=None, help='username')
 
+        parser.add_argument('-c', '--cache-only', action='store_true', default=False, 
+                            help='Retrieve the host inventory by cache only. Default is false.')
 
-        self.args = parser.parse_args()
+        parser.add_argument('-o', '--openshift-version', action='store', required=True,
+                            help='Specify the openshift account version.')
```

Comment:
"account version" ?

I see what you are going for, but I don't think account version is correct.

I think just openshift version is probably good.